lib/sandbox/pm-require.js

```diff
@@ -0,0 +1,91 @@
+const MODULE_KEY = '__module_obj', // why not use `module`?
+    MODULE_WRAPPER = [
+        '(function (exports, module) {\n',
+        `\n})(${MODULE_KEY}.exports, ${MODULE_KEY});`
+    ];
+
+class PostmanRequireStore {
+    constructor (fileCache) {
+        this.fileCache = fileCache || {};
+    }
+
+    getResolvedPath (path) {
+        if (this.hasFile(path)) {
+            return path;
+        }
+    }
+
+    hasFile (path) {
+        return Boolean(this.fileCache[path]);
+    }
+
+    getFileData (path) {
+        return this.hasFile(path) && this.fileCache[path].data;
+    }
+}
+
+function createPostmanRequire (fileCache, scope) {
+    const store = new PostmanRequireStore(fileCache),
+        cache = {};
+
+    return function postmanRequire (name) {
+        const path = store.getResolvedPath(name);
+
+        // Any module should not be evaluated twice, so we use it from the
+        // cache. If there's a circular dependency, the partially evaluated
+        // module will be returned from the cache.
+        if (cache[path]) {
+            // Always use the resolved path as the ID of the module. This
+            // ensures that relative paths are handled correctly.
+            return cache[path].exports;
+        }
+
+        /* eslint-disable-next-line one-var */
```

Comment:
@codenirvana I don't see the use of this eslint rule. I want to use `const` because the value never changes, but I don't want to evaluate it if I can bail out early. What're your thoughts?